src/Model/Resolver/Products/DataPostProcessor/Attributes.php

```diff
@@ -414,6 +418,7 @@ public function process(
 
         $isSingleProduct = isset($processorOptions['isSingleProduct']) ? $processorOptions['isSingleProduct'] : false;
         $isCompare = isset($processorOptions['isCompare']) ? $processorOptions['isCompare'] : false;
+        $isCartProduct = isset($processorOptions['isCartProduct']) ? $processorOptions['isCartProduct'] : true;
```

Comment:
So if `isCartProduct` wasn't passed, we assume it's `true` by default?
I believe it's a mistake.

Reply:
Otherwise it's not working properly, for cartItems it's passing firstly true and then false if I make false by default

Reply:
Then the implementation is wrong because this tells that all products are cart products unless you pass false.
But you don't pass false ever.

Reply:
In DataPostProcessor it's false by default, but I'll look through again and try to change this